modules/lead/toolchain/grafeas.tf

```diff
@@ -21,7 +21,7 @@ module "certificate" {
 resource "helm_release" "grafeas" {
   name       = "grafeas-server"
   namespace  = var.namespace
-  repository = "./grafeas-chart/"
+  repository = "../../modules/lead/toolchain"
```

Comment:
why is this path going up and then back down to same path?

Reply:
The working directory when terraform tries to deploy the grafeas helm chart is stacks/environment-aws, which threw us off when we were trying to get this working because we also assumed that the working directory would be modules/lead/toolchain

Reply:
Check out this sample: https://github.com/liatrio/lead-terraform/blob/master/modules/lead/infrastructure/certmanager.tf#L5